zubhub_frontend/zubhub/src/App.js

```diff
@@ -15,10 +15,24 @@ const SearchResults = React.lazy(() => import('./views/search_results/SearchResu
 
 const Signup = React.lazy(() => import('./views/signup/Signup'));
 const Login = React.lazy(() => import('./views/login/Login'));
+
+const PasswordReset = React.lazy(() =>
+  import('./views/password_reset/PasswordReset'),
+);
+const PasswordResetConfirm = React.lazy(() =>
+  import('./views/password_reset_confirm/PasswordResetConfirm'),
+);
+const EmailConfirm = React.lazy(() =>
+  import('./views/email_confirm/EmailConfirm'),
+); 
+const PhoneConfirm = React.lazy(() =>
+  import('./views/phone_confirm/PhoneConfirm'),
+);
```

Comment:
This change doesn't fit the description of the PR. If I am not mistaken this is likely from some kind of rebase/merge gone wrong. Do you mind removing this change if this is so? if this is indeed a part of this PR, do you mind changing the PR description to mention what these changes do?